### PR TITLE
Enhanced readability and updated Debian package names

### PR DESCRIPTION
**Pull request by asheroto**
Fixed misspellings, grammar, unneeded line breaks, enhanced readability, updated requirements section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,47 +7,41 @@ functionality with your MEGA account via commands. It features **synchronization
...
-have support for regular expressions.
+The requirements are the same as those for the sdk (usually `cryptopp, zlib, sqlite3, cares, libuv, ssl, curl,sodium, readline` for platforms other than Windows. It is recommended to include `pcre` to have support for regular expressions.
 
-Also, in order to have support for thumbnails and previews,
-it is highly recommended to have `ffmpeg` (`libavcodec-dev libavutil-dev libavformat-dev libswscale-dev`) and `mediainfo`(`libmediainfo-dev + libzen-dev`) for media file attributes.
+In order to have support for thumbnails and previews, it is highly recommended to have `ffmpeg` (`libavcodec-dev libavutil-dev libavformat-dev libswscale-dev`) and `mediainfo`(`libmediainfo-dev + libzen-dev`) for media file attributes.
 
-* For convenience here is a list of packages for ubuntu 18.04: `autoconf libtool g++ libcrypto++-dev libz-dev libsqlite3-dev libssl-dev libcurl4-gnutls-dev libreadline-dev libpcre++-dev libsodium-dev libc-ares-dev libfreeimage-dev libavcodec-dev libavutil-dev libavformat-dev libswscale-dev libmediainfo-dev libzen-dev libuv1-dev`
+**For convenience, here is a list of packages for common distros:**
```
Comment (polmr) [inline README.md]: Windows is listed here as a distro when it's not

File: README.md [hunk 2]
```diff
@@ -72,49 +67,36 @@ won't work. You would need to `source /YOUR/PREFIX/etc/bash_completion.d/megacmd
...
 # Usage
 
-Before explaining the two ways of interaction, it is important to understand how 
-MEGAcmd works. When you login with MEGAcmd, your session, the list of synced folders,
-and some cache database are stored in your local home folder. MEGAcmd also stores
-some other configuration in that folder. Closing it does not delete those and 
-restarting your computer will restore your previous session (the same as megasync 
-won't ask for user/password once you restart your computer). 
+Before explaining the two ways of interaction, it is important to understand how MEGAcmd works. When you login with MEGAcmd, your session, the list of synced folders, and some cache database are stored in your local home folder. MEGAcmd also stores some other configuration in that folder. Closing it does not delete those and restarting your computer will restore your previous session (the same as megasync won't ask for user/password once you restart your computer). 
+
```
Comment (polmr) [inline README.md]: Extra line break that breaks the next line into a new paragraph (unwanted)

**Comment by polmr** [comment]
Thanks @asheroto , If you address those points, the changes could be merged I believe

**Comment by asheroto** [comment]
Okay, I think I got it done.

**Comment by asheroto** [comment]
@polmr Do I need to do a new pull request?

File: README.md [hunk 3]
```diff
@@ -6,47 +7,37 @@ functionality with your MEGA account via commands. It features **synchronization
...
 
-and for ubuntu 16.04: `autoconf libtool
-g++ libcrypto++-dev libz-dev libsqlite3-dev libssl-dev libcurl4-openssl-dev
-libreadline-dev libpcre++-dev libsodium-dev libc-ares-dev libfreeimage-dev
-libavcodec-dev libavutil-dev libavformat-dev libswscale-dev libmediainfo-dev libzen-dev`
+ * **Ubuntu 18.04**
+	 * `autoconf libtool g++ libcrypto++-dev libz-dev libsqlite3-dev libssl-dev libcurl4-gnutls-dev libreadline-dev libpcre++-dev libsodium-dev libc-ares-dev libfreeimage-dev libavcodec-dev libavutil-dev libavformat-dev libswscale-dev libmediainfo-dev libzen-dev libuv1-dev`
 
-For Windows, here is a bundle with all the 3rd party dependencies required to build:
-https://mega.nz/#!MkZmVazb!fffUvFV9pIQhr__dVipwWqFfvjDEEp0JUsT8WBKAbRg.
```
Comment (polmr) [inline README.md]: Now this is nowhere I'm afraid

**Comment by polmr** [comment]
> @polmr Do I need to do a new pull request?

No, we can use this one. Please add the Windows part somewhere as another commit if you don't mind :-)

File: README.md (hunk 3 above)
Comment (asheroto) [reply README.md]: There, how does it look now?

**Comment by asheroto** [comment]
I think we're good now.

**Comment by asheroto** [comment]
Can you check that MEGA Url for Windows?   I'm not sure it's the right URL...

**Comment by asheroto** [comment]
> Can you check that MEGA Url for Windows? I'm not sure it's the right URL...
> 
> On my machine it just pauses at the mega loading screen. Checked in Chrome & Firefox.

Nevermind, it just took a little bit to load and I wasn't patient enough.  👍

**Comment by polmr** [comment]
merged. Thanks @asheroto for your contribution !

**Comment by asheroto** [comment]
@polmr Glad to help!

File: README.md [hunk 4]
```diff
@@ -6,47 +7,41 @@ functionality with your MEGA account via commands. It features **synchronization
 See [`Usage Examples`](#usage-examples).
```
Comment (saeed28N) [inline README.md]: sax

**Pull request merged**